upgrade.sh

```diff
@@ -52,7 +52,7 @@ if [ "${OS}" == "darwin" ]; then
     fi
 fi
 
-oc patch deployment acm-custom-registry -n ${TARGET_NAMESPACE} --type=json -p '[{"op":"replace","path":"/spec/template/spec/containers/0/image","value":"'${CUSTOM_REGISTRY_REPO}'/acm-custom-registry:'${NEXT_SNAPSHOT}'"}]'
+oc patch catalogsource acm-custom-registry -n openshift-marketplace --type=json -p '[{"op":"replace","path":"/spec/image","value":"'${CUSTOM_REGISTRY_REPO}'/acm-custom-registry:'${NEXT_SNAPSHOT}'"}]'
```

Comment:
Does this change functionality for pre-2.5 upgrade steps, we still need this to work for all supported z streams, including those before MCE

Reply:
This does not change functionality for pre-2.5 upgrade steps. This should still work as expected for all supported z-streams